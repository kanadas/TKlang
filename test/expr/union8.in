//TODO propably should work
((\a b -> if a + b == 0 then 1 @ 'a' else 2 @ a + b) :: int -> int -> char + int + bool) 1 2

