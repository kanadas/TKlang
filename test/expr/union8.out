expr/union8.in
Type mismatch: [] and [(char+int)] in expression:
(\ a b -> if a + b == 0 then 1 @ 'a' else 2 @ a + b) :: int -> int -> char + int + bool
